atomic_reactor/odcs_util.py

```diff
@@ -60,6 +61,11 @@ def start_compose(self, source_type, source, packages=None, sigkeys=None, arches
                         these keys will be included in a compose.
         :param arches: list<str>, List of additional Koji arches to build this compose for.
                         By default, the compose is built only for "x86_64" arch.
+        :param multilib_arches: list<str>, List of Koji arches to build as multilib in this
+                        compose. By default, no arches are built as multilib.
+        :param multilib_method: list<str>, list of methods to determine which packages should
+                        be included in a multilib compose. By default, is none and must be
+                        set to non-default if multilib_arches is not empty.
```

Comment:
Is the wording about "must be set to non-default" still accurate?

Reply:
Fixed.